### PR TITLE
Request to review codes for Invest Section

### PR DESCRIPTION
**Pull request by sayantanigoswami1996**
Kindly find the below list of Invest section to review the codes : 
1. SaveSmart - Invest Tiles and Explore
2.FD and Crypto ( screen navigation is covered as this page would be revamped in future) - Invest Tiles and Explore
3.Sector Funds
4.Value Funds
5.52Wk High US and India
6.Bank and PSU Bonds
7.Equity Index 
8.ELSS Tax Saver
9 . ETF
10 . Digi Gold - Explore and Invest
11.Gilt Funds
12.Ultra Short
13. MF - Explore

### CODE REVIEW

File: AppLocators/DesktopWeb/HealthInsurancePostLoginLocators/HealthInsuranceLandingPageLocators.robot [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+${e_HI_stateField} =  Karnataka
+${e_HI_cityField} =  Bengaluru
+${e_HI_phoneNumLabel} =  Phone number
+${e_HI_completeAddress} =  Sri Lakshminarayana Mansion Tata Silk Farm,Jayanagar
+${e_HI_endorsementLink} =  Endorsement wording                   
+${e_HI_policyWordingLink} =  Policy wording                  
+${e_HI_generalExclusionLink} =  General exclusions                    
+${e_HI_cashlessHospitalLink} =  List of cashless hospitals
+${e_HI_coverTitle} =  Covers
+${e_HI_totalCoverDesc} =  Total cover\n₹10.0 L + ₹2.0 K Daily cash
```
Comment (nivethan-moolya) [inline AppLocators/DesktopWeb/HealthInsurancePostLoginLocators/HealthInsuranceLandingPageLocators.robot:18]: Is the "\n" new line? How are we using this as a text?

File: Resources/Utils/DesktopWeb/Common.robot [hunk 2]
```diff
@@ -13,137 +13,281 @@ Resource    ../../../AppLocators/DesktopWeb/InvestLocators/SaveSmartLocators.rob
...
-        ${fieldTextLen} =  Get Length  ${fieldText} 
-        #Log To Console  ${fieldText}
-        #Log To Console  ${fieldTextLen}
-        Run Keyword If    """${fieldText}""" != ''
-        ...     Repeat Keyword  ${fieldTextLen+1}  Press Keys  ${field}  \ue003
+    [Arguments]  ${field}
+    Sleep  500ms
+    ${fieldText} =  Get Value  ${field}
+    ${fieldTextLen} =  Get Length  ${fieldText}
+    Run Keyword If    """${fieldText}""" != ''
```
Comment (nivethan-moolya) [inline Resources/Utils/DesktopWeb/Common.robot:150]: Is this correct?

File: AppLocators/DesktopWeb/HealthInsurancePostLoginLocators/HealthInsuranceLandingPageLocators.robot (hunk 1 above)
Comment (sayantanigoswami1996) [reply AppLocators/DesktopWeb/HealthInsurancePostLoginLocators/HealthInsuranceLandingPageLocators.robot:18]: Yes , if the text comes on a two different line , \n is used as escape character and works as expected.

File: Resources/Utils/DesktopWeb/Common.robot (hunk 2 above)
Comment (sayantanigoswami1996) [reply Resources/Utils/DesktopWeb/Common.robot:150]: Yes, its already reviewed last time and works fine.

**Pull request closed without merging**